src/js/edu-benefits/components/military-history/MilitaryServiceReview.jsx

```diff
@@ -45,20 +48,12 @@ export default class MilitaryServiceReview extends React.Component {
                 <td>{displayDateIfValid(tour.dateRange.to)}</td>
               </tr>
               <tr>
-                <td>Service status:</td>
-                <td>{tour.serviceStatus.value}</td>
-              </tr>
-              <tr>
-                <td>Were you involuntarily called for active duty during this period?</td>
-                <td>{getLabel(yesNo, tour.involuntarilyCalledToDuty.value)}</td>
-              </tr>
-              <tr>
-                <td>Apply this service period to the benefit I'm applying for</td>
+                <td>Apply this service period to the benefit I’m applying for.</td>
                 <td>{tour.applyPeriodToSelected ? 'Yes' : 'No'}</td>
               </tr>
               {!tour.applyPeriodToSelected
                 ? <tr>
-                  <td>Please explain how you'd like this service period applied.</td>
+                  <td>Please explain how you’d like this service period applied.</td>
```

Comment:
If it looks ok in the browser, it should be fine. It's just easy to have character encoding issues.